src/json-structure.rst

```diff
@@ -282,6 +282,8 @@ Replication Settings
 | use_checkpoints (optional)     | Whether to use replication checkpoints      |
 |                                | or not.                                     |
 +--------------------------------+---------------------------------------------+
+| winning_revs_only (optional)   | Replicate only the winning revisions        |
```

Comment:
Should this end with a period for consistency?

Reply:
Good point. It would look better that way.